api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -644,5 +654,195 @@ public void getMemberships_shouldGetMembershipsAsOfADate() throws Exception {
 		assertFalse(membersAsOfDate.isEmpty());
 		assertTrue(membersAsOfDate.stream().anyMatch(m -> m.getStartDate().equals(dateToTest)));
 	}
-
+	
+	/**
+	 * @verifies {@link Cohort#getMemberships(Date)}
+	 */
+	@Test
+	@Verifies(value = "should not get matching memberships of a cohort as of a date", method = "getMemberships(Date)")
+	public void getMemberships_shouldNotGetMatchingMembershipsAsOfADate() throws Exception {
```

Comment:
Since these test `CohortMembership.java`, should I move them to their own test class `CohortMembershipTest.java`?